aptos-move/aptos-vm/src/aptos_vm.rs

```diff
@@ -2008,11 +2012,23 @@ impl AptosVM {
                 let output = VMOutput::empty_with_status(status);
                 (VMStatus::Executed, output, Some("state_checkpoint".into()))
             },
-            Transaction::ValidatorTransaction(txn) => {
+            Transaction::ValidatorTransaction(_txn) => {
                 fail_point!("aptos_vm::execution::validator_transaction");
-                let (vm_status, output) =
-                    self.process_validator_transaction(resolver, txn.clone(), log_context);
-                (vm_status, output, Some("validator_transaction".to_string()))
+                #[cfg(any(test, feature = "fuzzing"))]
+                {
+                    #![allow(unreachable_code)]
+                    let (_vm_status, _output) =
+                        self.process_validator_transaction(resolver, _txn.clone(), log_context);
+                    (
+                        _vm_status,
+                        _output,
+                        Some("validator_transaction".to_string()),
+                    )
+                }
+                #[cfg(not(any(test, feature = "fuzzing")))]
+                {
+                    unreachable!()
+                }
```

Comment:
this will be reachable in a few days :)

Reply:
So... are the rest of us supposed to stop work until then?  When it becomes reachable, update it.

Reply:
actually, how does validator transaction processing affect the build? it's supposed to be always-on.

Reply:
See https://github.com/aptos-labs/aptos-core/issues/11445.  Enumerator `ValidatorTransaction` has no values in the base case (no fuzzing or test), so `ValidatorTransaction::clone()` is guaranteed to fail and following code is dead code.  (Seems overly conservative, since you can't have a value to clone without any possible values, so if we can only get here with a `ValidatorTransaction` value, then this code must be unreachable, but that's Rust's choice.)

Reply:
@zekun000 let's have `DummyTopic1` always on then. It doesn't really hurt, right?

Reply:
fine to me, we can remove once we have real variant

Reply:
Fixing it in #11452